content/en/platform/corda/4.10/community/release-checksum-os.md

```diff
@@ -1,21 +1,921 @@
+
+* library-master-v1.1-beta-g6215ec8-87.jar (357 Kb / 365733 bytes)
+  * `f1b33486539b936c69cbb87f0452e416f4ae70e0` (sha1)
+  * `6ca9531c5f4cb5f0e2ee564dfbda8e25e4f36a19f1f07f9ff30148624a8383cb` (sha256)
+  * `bc9c9186c8aacdaca77fe6cbbd80dbeb` (md5)
+
+* library-master-v1.1-beta-g6215ec8-87.pom (423 bytes / 423 bytes)
+  * `c9787884f3b23556ca1e9bfda7c8dc5a9a47e048` (sha1)
+  * `297629e72496bd25e5fae6b60da88b869afe679075bd8623d39113c373d35d80` (sha256)
+  * `5f7a3fedb6073e32fca45a212aead7df` (md5)
+
+* quasar-core-0.7.15_r3.jar (1 Mb / 1297443 bytes)
+  * `ee984e4eece6cba4ada6ceca04c491efc2a20eba` (sha1)
+  * `b289deb22c7137e7408a78b0d7c27b56f92ca85539606e718656a061cd404640` (sha256)
+  * `1aa39dd0702ba4cc9e29ad26ea52dd68` (md5)
+
+* quasar-core-0.7.15_r3-javadoc.jar (1 Mb / 1193680 bytes)
+  * `b782cfb03a413a326c306ab9205c09003c86db03` (sha1)
+  * `2aa1f24e99b5ba6756de79bf7a7fd83315b38ecbca6ef54607614a9ebc16d415` (sha256)
+  * `bed80b8a25e2576202ef80911fcaf08e` (md5)
+
+* quasar-core-0.7.15_r3.pom (2 Kb / 2660 bytes)
+  * `e4d7e5bf7696a2728c7824ea4c31dc7810486fb7` (sha1)
+  * `bf77586ba436eea1a66ddbd418972df20f36bca8a2c525d904bb332d075f649b` (sha256)
+  * `efa9bf3f3000cdf61c57340d2579f9fd` (md5)
+
+* quasar-utils-5.0.17.jar (31 Kb / 32472 bytes)
+  * `515fa7c2bace6d138814f872fd6089cdf98d6c13` (sha1)
+  * `536073280c1aa8c66b812cd34f69951b847bfffcb9f1487cc19a4d808d859abb` (sha256)
+  * `3860e1f1b58fc79ac6a5399f9401ecbb` (md5)
+
+* quasar-utils-5.0.17-javadoc.jar (261 bytes / 261 bytes)
+  * `e0bdc5633a66f27890d73d4f4e507fc5dbd12946` (sha1)
+  * `eeea0d2b007ad2a261c3dfaf1ff6afbf70b0f0a03d64a00c14582851fe4d988f` (sha256)
+  * `bfe4fa7cf85c8bf08c649babc4aa4a2d` (md5)
+
+* quasar-utils-5.0.17.pom (1 Kb / 1029 bytes)
+  * `f60a4cc8eb073310725b0d646d128593a9336e6a` (sha1)
+  * `0eebf8bef30f51602abd3b36c16403dd49e84e0c3ca928c60cd5e6899036c4ff` (sha256)
+  * `e9bfbc7d2811c0b25a8d660d5bcf6833` (md5)
+
+* quasar-utils-5.1.1.jar (31 Kb / 32307 bytes)
+  * `8bd2fda425ed0f49f13bc66f3bdd971c29127fc4` (sha1)
+  * `b28b5dc748b1add662cea0ee1e953837967003af6173107ec473308eb77893ef` (sha256)
+  * `8ad624cbfd6b9cbae90cf9e88ed2e6d9` (md5)
+
+* quasar-utils-5.1.1-javadoc.jar (261 bytes / 261 bytes)
+  * `714ee1621d42b843c5de8f83ab82ef6821b0fe7d` (sha1)
+  * `c6511c409ab40bb17194bee8adc4e365fefea56765f025b61684be8a44ac392f` (sha256)
+  * `218f273c441340779650bc7918e501c5` (md5)
+
+* quasar-utils-5.1.1.pom (1 Kb / 1029 bytes)
+  * `c24b2d08defcd9f263df6967ec8f45e655c811fa` (sha1)
+  * `881061e09c44fd0df936f5ec0cd4d1b73fb2e642b469dc9656787324699ff7a9` (sha256)
+  * `24924975cfc3a34310c850f249f5d1ba` (md5)
 
-Before you install the downloaded files, verify the integrity of each file to ensure it has not been tampered with or corrupted during the download. You can do this by calculating the **checksum** of your downloaded file, then comparing its value to the checksum value provided below.
 
 ## Checksums of Corda Community Edition 4.10 release files
```

Comment:
```suggestion
## Checksums of Corda Community Edition 4.10 release packs
```

?

Reply:
The new terminology is "packs" - although for a patch there is only one rather than multiple packs in a release.